codegen/core/src/main/scala/com/lightbend/akkasls/codegen/ModelBuilder.scala

```diff
@@ -66,7 +66,10 @@ object ModelBuilder {
       override val fqn: FullyQualifiedName,
       override val entityType: String,
       state: State)
-      extends Entity(fqn, entityType) {}
+      extends Entity(fqn, entityType) {
+    val abstractEntityName = "Abstract" + fqn.name
```

Comment:
We can move this to `Entity` so we can ask for the abstract name whenever we need it. Useful for both Scala and Java and for all types of entities.

Reply:
Let's do that in separate PR?